packages/next/build/webpack/config/blocks/css/index.ts

```diff
@@ -395,6 +384,21 @@ export const css = curry(async function css(
     )
   }
 
+  fns.unshift(
+    loader({
+      oneOf: [
+        {
+          // Impossible regex expression
+          test: /a^/,
+          loader: 'noop-loader',
+          // Record the number of entries we need to remove if built-in CSS
+          // support is disabled
+          options: { __next_css_remove: fns.length + 1 },
```

Comment:
Maybe we should get rid of that at all and instead add a `Symbol("next css")` to all `fns` above and filter rules with that symbol.

Reply:
I had that idea too, but I didn't know where to add such a symbol. The loaders are of type `RuleSetRule` and don't allow additional attributes (it leads to a compile error, and AFAIK webpack also throws an error at run-time). Let me know what you envisage and I will update the PR.